lib/matplotlib/cm.py

```diff
@@ -325,6 +325,15 @@ def set_clim(self, vmin=None, vmax=None):
             self.norm.vmax = colors._sanitize_extrema(vmax)
         self.changed()
 
+    def get_alpha(self):
```

Comment:
Would leave a comment (not necessarily in the docstring) like "this method is intended to be overridden by Artist subclasses".